zowe/zowe-cli-secrets-for-kubernetes
# Export credential manager API from CLI plugin

Resolves #8 

Things addressed in this PR:

- Export credential manager API from CLI plugin
- Updated dependencies to latest versions
- Changed `author` names in packages to `Zowe` (suggested by @JillieBeanSim :) )
- Optimized bundling times with `tsup` bundler in combination with `turbo`! now minimizes extension with `terser`.
- When activating extension, a constructor for the credential manager can be used for initializing
- Adjusted the `.editorconfig` to use 4 spaces instead of 2 (was seeing some warnings of this on build command
- Added support for lifecycle methods `postInstall` and `preUninstall`

to test:
run `npm install && npm run build`, view the `packages/vscode/dist/extension.js` file and notice how it's now minimized, this should run without errors.

run `npm run package` this should bundle, minimize, and pacakge the CLI and `.vsix` file in the `dist` directory.
